examples/log/src/main.rs

```diff
@@ -32,13 +32,13 @@
 //! cargo run --release -- --bootstrappers 0@127.0.0.1:3000 --me 1@3001 --participants 0,1,2,3 --storage-dir /tmp/commonware-log/1
 //! ```
 //!
-//! # Participant 2
+//! ## Participant 2
```

Comment:
Would you mind making the accompanying fix in the `README` (has the same issue)?